ironfish/src/network/peers/addressManager.ts

```diff
@@ -2,24 +2,149 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 import { HostsStore } from '../../fileStores'
+import { FileSystem } from '../../fileSystems'
+import { ArrayUtils } from '../../utils'
+import { Peer, PeerList } from '..'
+import { ConnectionDirection, ConnectionType } from './connections'
 import { PeerAddress } from './peerAddress'
 
 /**
  * AddressManager stores the necessary data for connecting to new peers
  * and provides functionality for persistence of said data.
  */
 export class AddressManager {
-  hosts: HostsStore
+  hostsStore: HostsStore
 
-  constructor(hostsStore: HostsStore) {
-    this.hosts = hostsStore
+  constructor(files: FileSystem, dataDir?: string) {
+    this.hostsStore = new HostsStore(files, dataDir)
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    this.hostsStore.load()
```

Comment:
I think this lint error should be addressed rather than disabled, since there could be bugs related to other code executing before the loading has finished.